lamarr/roster.json

```diff
@@ -96,7 +96,9 @@
       "github": "GabrielleMonaen",
       "materials": [
         {
-          "type": "cv"
+          "type": "cv",
+	  "type": "video",
+	  "url": "x35oaP72rFg"
         }
```

Comment:
Unfortunately, they cannot be combined, we need 2 objects inside the materials array

Reply:
Created a new pull request with the correct values.